custom_components/econet300/sensor.py

```diff
@@ -151,27 +148,41 @@ def create_mixer_sensor_entity_description(key: str) -> EconetSensorEntityDescri
     return entity_description
 
 
-def create_mixer_sensors(coordinator: EconetDataCoordinator, api: Econet300Api):
+# Dynamically generate SENSOR_MIXER_KEY
+SENSOR_MIXER_KEY = {
```

Comment:
this should go to const.py, also would advise to rename attributre  (the point is to make it plural, should enhance readabilyti)

```suggestion
AVAILABLE_SENSOR_MIXER_KEYS = {
```
or 
```suggestion
SENSOR_MIXER_KEYS = {
```